apps/site/pages/docs/building-sections/overrides/override-component.mdx

```diff
@@ -0,0 +1,242 @@
+
+![overrides-PDP-buy-button-component](https://vtexhelp.vtexassets.com/assets/docs/src/overrides-buy-button-open-cart___a5853a88238fa31610a93e8a1333bb00.gif)
+
+For this guide we will create a custom Buy Button that once you click on it, it displays an `alert` for the customer.
+
+> ⚠️ Using overrides does not significantly impact performance. FastStore's handling of props and design tokens is designed for good performance. However, your customizations may introduce additional work, so consider overall store performance by checking Lighthouse reports on GitHub logs and Pull Requests.
+
+---
+
+## Before you begin
+
+Make sure your `@faststore/core` package has the 3.x version or above. If you need to update
+it, refer to this [release note](https://github.com/vtex/faststore/releases/tag/v3.0.0).
+
+--- 
+
+## Step by step
+### Step 1 - Setting up the component file
+
+1. After choosing a native section to be customized from the [list of available native sections](https://developers.vtex.com/docs/guides/faststore/building-sections-list-of-native-sections), open your FastStore project in any code editor of your preference.
+2. Go to the `src` folder, create the `components` folder, and inside it, create the `sections` folder. You can run the following command to create them:
+
+> ℹ️ The naming of the `sections` folder is arbitrary, as overrides can be created in any file, since the import is made in the `src/components/index.tsx` file. 
+
+<Tabs items={['macOS and Linux', 'Windows']}>
+  <Tab>```bash copy mkdir src/components src/components/sections ```</Tab>
+  <Tab>```bash copy mkdir src\components src\components\sections ```</Tab>
+</Tabs>
+
+3. In the `sections` folder, create a new file for your customized section. For example, create a new file named `ProductDetailsWithCustomButton.tsx` under the `src/components/sections` directory.
+
+4. Open the `ProductDetailsWithCustomButton.tsx` file and update it with the following code:
+
+```tsx filename="ProductDetailsWithCustomButton.tsx" {8-9}
+import { ProductDetailsSection, getOverriddenSection } from '@faststore/core'
+
+const ProductDetailsWithCustomButton = getOverriddenSection({
+  Section: ProductDetailsSection,
+  components: {
+    BuyButton: { Component: CustomBuyButton },
+  },
+})
+```
+
+#### Step 2 - Overriding the component
+
+1. Choose a component to override from the [list of overridable components of each native section](https://developers.vtex.com/docs/guides/faststore/building-sections-list-of-native-sections). 
+In this example, we are overriding the `BuyButton` component in the `ProductDetails` section.
+
+2. In the `components` folder, if you don't have already the `index.tsx` file, create the file and add the following:
+
+```tsx filename="components/index.tsx" copy
+import CustomBuyButton from "./sections/CustomBuyButton";
+
+export default { CustomBuyButton };
```

Comment:
The section here is the `ProductDetailsWithCustomButton` instead of `CustomBuyButton`
```suggestion
import ProductDetailsWithCustomButton from "./sections/ProductDetailsWithCustomButton";

export default { ProductDetailsWithCustomButton };
```